### DISCUSSION TITLE
buddhism/sense/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#47] [General]
# buddhism/sense/

https://shanenull.com/buddhism/sense/

**Comment by shane0** [comment]
![](https://user-images.githubusercontent.com/1454458/280515072-dcaaa28b-1d51-4abb-88c8-6b38689eaf28.png)

**Comment by shane0** [comment]
### One Mind Without Scatteredness

https://www.ctworld.org.tw/Buddhist%20e-Books/CTM/CTM233GM/index.html

**Comment by shane0** [comment]
aka six pollutants

**Comment by shane0** [comment]
week 9 story about turtle and otter 

The Buddha told the shramana, 
“Cultivators should know how to protect 
themselves, just like this turtle. The 
numberless kalpas of life and death we 
spend in samsara are all because our eyes, 
ears, nose, tongue, body, and mind act 
improperly in response to conditions. When 
the eyes see attractive forms, we give rise 
to craving and delusive thoughts. When the 
ears hear agreeable sounds or praise, we feel 
delighted; but if they hear others malign us, 
we give rise to afflictions instead. The six 
senses are constantly attaching to conditions 
and misconstruing reality, taking external 
phenomena to be real and substantial. This is 
why we cannot realize the Way.”

Then, the Buddha spoke a gatha: “Draw 
in the six senses like a turtle, guard the 
mind like a city. Use wisdom to fight the 
maras, with victory comes freedom from 
afflictions.” In our cultivation, we should be 
like the turtle, “drawing in the six senses” 
to safeguard the wisdom life of our Dharma 
body.  The  eyes must  look  inward,  and the 
ears must listen inward; “listen” and “look” 
inward to our intrinsic awareness.
The ancients said, “Cork the mouth like 
a bottle, guard the mind like a city.” It is 
easy for us to speak wrongly with the tongue 
and for the consciousness to raise distorted 
views. Thus, we must “guard the mind like 
a city,” like a garrison and its commander 
defending the city gate and be alert yet still.

Wherever we are, that is where the 
mind is. In terms of meditation, that means 
not raising wandering thoughts or falling 
into drowsiness, there is only this very 
mind always present. In terms of walking, 
the mind is present in the walking. In terms 
of performing a task, the mind is there 
performing  the  task.  In terms  of  reciting 
buddhas’ names, the mind is present with the 
reciting. In terms of reciting a sutra, the mind 
is there reciting the sutra. But as soon as we 
are done doing, nothing remains in the mind; 
as soon as we are done speaking, nothing 
remains in the mind—after reciting a sutra, 
we put it down as if we had never recited it. 
The mind is always clear and lucid—in this 
way, we will be able to realize the state of 
non-arising and achieve bodhi. This is the 
sincerest [way of life].

定慧不二 心想事成

With non-duality of samadhi and 
prajna, actualize the wishes of 
the heart.

I hope that everyone will apply this 
teaching. The past is already gone, so start 
from this very minute, this very second. 
Keep the mind free of scatteredness and 
distorted views, and always be clear, lucid, 
unmoving in suchness, and ever-present 
with  the  knowing of  awareness—stillness 
and awareness must  be one. “Stillness” is 
to be unmoving; “awareness” is to be clear 
and lucid; “stillness” is shamatha (calm 
abiding); “awareness” is vipashyana (insight 
contemplation).  Succeeding  in these two 
practices is to simultaneously abide in 
samadhi and prajna where they are non-dual. 
This matter is entirely up to ourselves. 
We must realize the principle and apply 
it in practice. Having understood this, 
we cultivate with this very mind, never 
departing from its present awareness. Then, 
we will accomplish the Way in this life.

**Comment by shane0** [comment]
Form, sound, smell, taste, touch or dharmas

 The six sense objects, also known as the six pollutants or six dusts I. They are the objects perceived by the six sense

eyes (form), ears (sound), nose (smell), tongue (taste), body (touch)and mind (dharmas).
